port/common/omrsysinfo.c

```diff
@@ -936,3 +936,49 @@ omrsysinfo_cgroup_is_memlimit_set(struct OMRPortLibrary *portLibrary)
 {
 	return FALSE;
 }
+
+/**
+ * Gets the file pointer by opening the file passed as a param fileName
+ *
+ * @param[in] portLibrary pointer to OMRPortLibrary
+ *
+ * @param[in] subsystemFlag for checking the index of the subsystem
+ *
+ * @param[in] fileName pointer to cgroup subsystem filename
+ *
+ * @param[in] file pointer to the file pointer
+ *
+ * @return 0 on success and OMRPORT_ERROR_SYSINFO_CGROUP_UNSUPPORTED_PLATFORM on failure
+ */
+int32_t
+omrsysinfo_cgroup_get_handle_subsystem_file(struct OMRPortLibrary *portLibrary,  uint64_t subsystemFlag, const char *fileName, FILE **file)
+{
+	return OMRPORT_ERROR_SYSINFO_CGROUP_UNSUPPORTED_PLATFORM;
+}
+
+/**
+ * Gets the list of subsystems available
+ *
+ * @param[in] portLibrary pointer to OMRPortLibrary
+ *
+ * @return OMRCgroupEntry struct pointer of the linked list if available, NULL otherwise
+ */
+struct OMRCgroupEntry *
+omrsysinfo_get_cgroup_subsystem_list(struct OMRPortLibrary *portLibrary){
```

Comment:
nit: formatting (`{` should be on the next line).

Reply:
Thank you for the review @keithc-ca. Addressed this comment and made changes accordingly